<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePlayer</class>
 <widget class="QMainWindow" name="SimplePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="middle_layout">
      <item>
       <widget class="QWidget" name="VideoFrame" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <zorder>widget</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="channels_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Channels</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="PlayButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/play.png</normaloff>icons/play.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StopButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/stop.png</normaloff>icons/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="VolumeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/volume.png</normaloff>icons/volume.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volumeslider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fullscreen">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/full_in.png</normaloff>icons/full_in.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="seaprator_full">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>icons/vertical_separator.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="download_icon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/download.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="download_speed">
        <property name="text">
         <string>0 kbps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="upload_icon">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/upload.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="upload_speed">
        <property name="text">
         <string>0 kbps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Separator_icon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/vertical_separator.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="users">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Users Online : 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hidebutton">
        <property name="text">
         <string>hide channels</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="openfile"/>
    <addaction name="Quit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMedia"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpenLocal">
   <property name="text">
    <string>Open local file</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string> Add Chnnels</string>
   </property>
  </action>
  <action name="openfile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SimplePlayer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
